firebase-installations/src/main/java/com/google/firebase/installations/Utils.java

```diff
@@ -43,7 +48,12 @@ public boolean isAuthTokenExpired(PersistedInstallationEntry entry) {
 
   /** Returns current time in seconds. */
   public long currentTimeInSecs() {
-    return TimeUnit.MILLISECONDS.toSeconds(System.currentTimeMillis());
+    return TimeUnit.MILLISECONDS.toSeconds(currentTimeInMillis());
+  }
+
+  /** Returns current time in milliseconds. */
+  public long currentTimeInMillis() {
+    return System.currentTimeMillis();
```

Comment:
I probably asked this before:
Any chance we can inject a timesource here or any instance that can be overwritten in different environments?
Not important at all.
But if we plan to do something about this, then maybe add a comment or TODO.

Reply:
Correct, I have a open bug & PR addressing this request. https://github.com/firebase/firebase-android-sdk/pull/1333
I`ll address that PR shortly after submitting this.

Reply:
Just seeing this. Some comments above may allude to this ...